lib/Runtime/ByteCode/ByteCodeSerializer.cpp

```diff
@@ -3298,6 +3327,63 @@ class ByteCodeBufferReader
         return current;
     }
 
+    const byte * ReadPropertyIdOfFormals(const byte * current, FunctionBody * function)
+    {
+#ifdef BYTE_CODE_MAGIC_CONSTANTS
+        int constant;
```

Comment:
Done.

---
In reply to: [67773061](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67773061) [](ancestors = 67773061)